lens/sqlrepo/tstracker/chainbs.go

```diff
@@ -0,0 +1,131 @@
+package tstracker
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"sync"
+
+	pgchainbs "github.com/filecoin-project/go-bs-postgres-chainnotated"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/lib/blockstore"
+	logging "github.com/ipfs/go-log/v2"
+)
+
+var log = logging.Logger("tipset-tracker")
+
+type TrackingChainstore interface {
```

Comment:
What is the benefit of defining this interface? Can we simply rename `tcs` => `TrackingChainstore` and return a pointer to it

Reply:
@willscott note https://github.com/filecoin-project/sentinel-visor/pull/410#issue-586669973 before addressing: you might get away with `rm -r lens/sqlrepo/tstracker` entirely